packages/graphql/tests/integration/update.int.test.ts

```diff
@@ -1322,6 +1322,98 @@ describe("update", () => {
         }
     });
 
+    test("should update an Edge property in a one to one relationship", async () => {
+        const session = driver.session();
+
+        const typeDefs = `
+          type Product {
+             id: ID
+             color: Color! @relationship(type: "OF_COLOR", direction: OUT, properties: "OfColorProperties")
+           }
+
+           interface OfColorProperties @relationshipProperties {
+               test: Boolean
+           }
+
+           type Color {
+             name: String
+             id: ID
+           }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        const productId = generate({
+            charset: "alphabetic",
+        });
+
+        const colorId = generate({
+            charset: "alphabetic",
+        });
+
+        const query = `
+            mutation {
+                updateProducts(
+                  where: { id: "${productId}" }
+                  update: {
+                      color: {
+                          update: {
+                              edge: {
+                                  test: true
+                              }
+                          }
+                      }
+                  }
+                ) {
+                    products {
+                        id
+                        colorConnection {
+                            edges {
+                                test
+                            }
+                        }
+                    }
+                }
+              }
+        `;
+
+        try {
+            await session.run(
+                `
+                    CREATE (product:Product {name: "Pringles", id: $productId})
+                    CREATE (color:Color {name: "Yellow", id: $colorId})
+                    MERGE (product)-[:OF_COLOR { test: false }]->(color)
+            `,
+                {
+                    productId,
+                    colorId,
+                }
+            );
+
+            const gqlResult = await graphql({
+                schema: neoSchema.schema,
+                source: query,
+                variableValues: {},
+                contextValue: { driver, driverConfig: { bookmarks: session.lastBookmark() } },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+
+            expect((gqlResult?.data?.updateProducts.products as any[])[0]).toMatchObject({
+                id: productId,
+                colorConnection: {
+                    edges: [
+                        {
+                            test: true,
+                        },
+                    ],
+                },
+            });
+        } finally {
+            await session.close();
+        }
+    });
```

Comment:
We generally put new integration tests which cover specific GitHub Issues in the `tests/integration/issues` directory (see existing examples in https://github.com/neo4j/graphql/tree/master/packages/graphql/tests/integration/issues).

Would you mind moving this into there? Thanks!

Reply:
I moved it :)